static/locales/en-US.yaml

```
@@ -884,17 +876,29 @@ Video:
       reversing playlists: reversing playlists
       shuffling playlists: shuffling playlists
       looping playlists: looping playlists
-  Stats:
-    Video statistics are not available for legacy videos: Video statistics are not available for legacy videos
-    Video ID: Video ID
-    Resolution: Resolution
-    Player Dimensions: Player Dimensions
-    Bitrate: Bitrate
-    Volume: Volume
-    Bandwidth: Bandwidth
-    Buffered: Buffered
-    Dropped / Total Frames: Dropped / Total Frames
-    Mimetype: Mimetype
+  Player:
+    TranslatedCaptionTemplate: '{language} (translated from "{originalLanguage}")'
+    Theatre Mode: Theatre Mode
```

in en-US this would be
```suggestion
    Theatre Mode: Theater Mode
```